api/src/main/proto/optimizer.proto

```diff
@@ -76,16 +76,33 @@ service Optimizer {
 
     // commands/queries to OASIS
     rpc startOptimization (StartOptimizationCommandDTO) returns (StartOptimizationResponseDTO) {}
```

Comment:
Another note and I think I can come up new issues related to this. 
For cloud mode, for visibility to the client, we may want to pass as much information about optimizer as possible since client will never be able to see it. I think this means a std-out or st-error stream to client to take if they need it. I try it out a bit with a new register Log and passing back information that we write into the console. However, I running into some problem with the channel with kotlin so I decide to ditch it for mow.

The other thing on the client side for cloud. We can have current UI as a driver for user of API and help them skip the part writing the same/similar code again and put more value on the FakePlugin that we only used for stub. I think this can avoid future c/p. This is just an idea and I don't have anything really solid for now. However, I got this when I look at the refactoring on ansys when I add volition api to that.

Reply:
This also means more service/business logic treatment on the current fake plugin

Reply:
Agreed, I think logically it may help as we speculate in that way to come up something for user who not exactly interested using the api. Simply describe it as a middle man bridging the gap between a user already have their `evaluate(input):output`, which itself can be big and completed and 90% of work at this point adding a new support to a new tool.